vulnerabilities/models.py

```diff
@@ -531,8 +531,41 @@ class Package(PackageURLMixin):
         to="Vulnerability", through="PackageRelatedVulnerability"
     )
 
+    package_url = models.CharField(
+        max_length=1000,
+        null=False,
+        help_text="The Package URL for this package.",
+        db_index=True,
+    )
+
+    plain_package_url = models.CharField(
```

Comment:
I am not in love with the `plain_package_url` name but this makes sense for now. We will need to find a better way.
I will merge for now!